utils.py

```diff
@@ -102,11 +102,9 @@ def get_file(title):
 
 def title():
     settings = sg.UserSettings()
-    title = TITLE+settings['-company-']
-    title += '@'+settings['-server-']
-    title += " "+VERSION
-    return title
-
+    company = settings.get('-company-', 'Unbekannt')
+    server = settings.get('-server-', 'unbekannt')
+    return f'{TITLE}{company}@{server} {VERSION}'
```

Comment:
what is the rationale behind this refactoring? Why 'Unbekannt'?

Reply:
On a new installation, `settings['-company-']` evaluates to `None` and `str+None` gives an exception. Therefore, I'm using `.get()` with a `str` as default value. We could use something else.

The second change, i.e. using an f-string instead of multiple string concatenations improves readability and is the modern Python approach for string formatting. Adding strings with `+` has always been an anti-pattern in Python because it can be an incredible performance issue if many strings are added. The (old) pattern for adding many strings is `''.join([s1, s2, s3, ...])`. This is magnitudes faster than adding multiple strings because it happens completely in the fast C-implementation and building an array is also much much faster than adding strings. Basically, it's O(1) vs. O(n). For this small string the performance doesn't matter, but apart from better readability one should avoid the anti-pattern even for small numbers of strings, because other people will understandably simply copy the anti-pattern when they need to build a string from thousands of strings.

I guess with the usage of an f-string the first change could be reverted because an f-string will interpolate a `None` value to `'None'` instead of raising an exception.

Reply:
OK, understand. Then I would prefer the 'Unbekannt' version.